engine/core/src/main/java/org/datacleaner/configuration/RemoteServerConfigurationImpl.java

```diff
@@ -56,5 +87,103 @@ public RemoteServerData getServerConfig(String serverName) {
         return null;
     }
 
+    @Override
+    public RemoteServerState getActualState(String remoteServerName) {
+        if (serverStatusTask == null) {
```

Comment:
`scheduledTask()` makes this check, so I guess there's no need to repeat it (same below)